test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/concurrent/ConcurrentHashMapTest.scala

```diff
@@ -194,6 +194,34 @@ class ConcurrentHashMapTest extends MapTest {
         assertTrue(s"missing value '$value'", encounteredValues.contains(value))
     }
   }
+
+  @Test def addOnKeySetView(): Unit = {
+    val map = factory.empty[String, Int]
+    val keySet = map.keySet(0)
+    assertNull(map.get("ONE"))
+    assertTrue(keySet.add("ONE"))
+    assertEquals("Adding a new key adds the default value.", 0, map.get("ONE"))
+
+    assertFalse(keySet.add("ONE"))
+    assertEquals("Adding an existing key do not alter the value", 0, map.get("ONE"))
+
+    assertEquals(0, map.put("ONE", 42))
+    assertEquals("Putting alter the value", 42, map.get("ONE"))
```

Comment:
Nit: alter*s*

Reply:
Fixed in https://github.com/scala-js/scala-js/pull/4367/commits/873d05ed0a4fd4c60f549c666da8f20abf3515ef